engine/src/main/java/engine/object/TokenManager.java

```diff
@@ -0,0 +1,14 @@
+package engine.object;
+
+public class TokenManager {
+
+    private int currentId;
+
+    public TokenManager() {
+        this.currentId = 0;
+    }
+
+    public int getNewId() {
```

Comment:
For classes like TokenManager names look like:

nextId();

So getNewId() -> nextId()